### ISSUE TITLE
[Text analytics] Missing enums in HealthcareEntityRelation and HealthcareEntityCategory

### ISSUE THREAD

**Issue by justinqquall** [Azure/azure-sdk-for-python#27406]
- **Package Name**: azure-ai-textanalytics
- **Package Version**:  5.2.1
- **Operating System**:
- **Python Version**: 3.10
**Describe the bug**
The following enums are received from the API but not included in the list of enums for `HealthcareEntityRelation`
```
    COURSE_OF_EXAMINATION = "CourseOfExamination"
    COURSE_OF_TREATMENT = "CourseOfTreatment"
    EXAMINATION_FINDS_CONDITION = "ExaminationFindsCondition"
    BODY_SITE_OF_CONDITION = "BodySiteOfCondition"
    BODY_SITE_OF_TREATMENT = "BodySiteOfTreatment"
```
The following enums are received from the API but not included in the list of enums for `HealthcareEntityCategory`
```
    COURSE = "Course"
```
**To Reproduce**
Submit a document to healthcare analytics API that would result in extraction of the relations and relation categories described above.
**Expected behavior**
The client enum classes should include all enums by the API. If a downstream system relies on the enums of the client naively, it will crash.
**Screenshots**
**Additional context**
This happens when using model_version `2022-08-15-preview`. Does not appear to happen with older model versions. Probably a pre-release issue but please keep this in mind for next sdk release

**Comment by kristapratico** [comment]
Thanks @justinqquall, I'll take a look and get back to you asap.

**Comment by kristapratico** [comment]
@peytonfraser these enum values were not defined in the swagger spec for [2022-05-01](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cognitiveservices/data-plane/Language/stable/2022-05-01/analyzetext.json) and so the SDKs missed exposing them in the GA release. Can you provide the known list of values for these two enums?

**Comment by justinqquall** [comment]
Thank you, Krista!  Much appreciated.  Hope you’re having a nice week.

**Comment by kristapratico** [comment]
Adding @aurghob, can you share all the known values for these two enums? 

https://github.com/Azure/azure-rest-api-specs/blob/bdb63328ebe9214f0e513c0073ae7c97c6d4844f/specification/cognitiveservices/data-plane/Language/stable/2022-05-01/analyzetext.json#L825

https://github.com/Azure/azure-rest-api-specs/blob/bdb63328ebe9214f0e513c0073ae7c97c6d4844f/specification/cognitiveservices/data-plane/Language/stable/2022-05-01/analyzetext.json#L919

We need to update the swagger to include these and if we can, also update 2022-10-01-preview so we can expose these enums in the next SDK release.

**Comment by aurghob** [comment]
Hi,
Thanks for bringing this up. We will discuss these with the team that works on the model to make sure there are no other discrepancies before we move forward.

**Comment by kristapratico** [comment]
The enums have been updated in this PR (#30194) and will be available in the 5.3.0 release of the library.
